stl/inc/algorithm

```diff
@@ -9228,8 +9289,29 @@ namespace ranges {
 #endif // __cpp_lib_concepts
 #endif // _HAS_CXX17
 
+template <class _FwdIt, class _Pr>
+_INLINE_VAR constexpr bool _Is_min_max_optimization_safe =

```

Comment:
We are usually appending `_v` to variable templates

Reply:
Ugh, please stop.